docs/README.md

```diff
@@ -113,17 +114,11 @@ predicate types. For example, one predicate type might require an "or" between
 artifact IDs, while another requires an "and." This difference would add
 complexity and confusion.
 
-## Footnotes
+[^1]: This is compatible with the [SLSA Attestation Model].
 
-\[1]: The `expected_command` is only a warning, and `inspections` require
-running external commands which is infeasible in many situations.
+[^2]: The `expected_command` is only a warning, and `inspections` require running external commands which is infeasible in many situations.
 
-\[2]: That said, we strongly recommend against keying a database purely by
-content hash. The reason is that such databases quickly run into scaling issues,
-as explained in
-[Building Secure and Reliable Systems](https://static.googleusercontent.com/media/landing.google.com/en//sre/static/pdf/Building_Secure_and_Reliable_Systems.pdf#page=364),
-Chapter 14, page 328, "Ensure Unambiguous Provenance." Instead, we recommend
-keying primarily by resource name, in addition to content hash.
+[^3]: That said, we strongly recommend against keying a database purely by content hash. The reason is that such databases quickly run into scaling issues, as explained in [Building Secure and Reliable Systems](https://static.googleusercontent.com/media/landing.google.com/en//sre/static/pdf/Building_Secure_and_Reliable_Systems.pdf#page=364), Chapter 14, page 328, "Ensure Unambiguous Provenance." Instead, we recommend keying primarily by resource name, in addition to content hash.
```

Comment:
Why would this fix the problem? A quick explanation might help 🙂

Reply:
Think this should be resolved as part of #127

Reply:
Great, just make a note there, pls, thx 🙂

Reply:
done!